src/builders.ts

```diff
@@ -506,12 +506,18 @@ export async function makeContractCall(txOptions: ContractCallOptions): Promise<
   );
 
   if (options?.validateWithAbi) {
-    if (options?.network) {
-      const abi = await getAbi(options.contractAddress, options.contractName, options.network);
-      validateContractCall(payload, abi);
+    let abi;
```

Comment:
I think this is `let abi: any;`, I'm surprised the linter is allowing it. Unless typescript can infer this now.
`let abi: ClarityAbi` should be safer